internal/deviations/deviations.go

```diff
@@ -645,4 +650,6 @@ var (
 	routePolicyUnderAFIUnsupported = flag.Bool("deviation_route_policy_under_afi_unsupported", false, "Set true for device that does not support route-policy under AFI/SAFI, default is false")
 
 	interfaceRefConfigUnsupported = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
+
+	skipNtpNonDefaultVrf = flag.Bool("deviation_skip_ntp_non_default_vrf", false, "Set true to skip the non default vrf ntp case")
```

Comment:
With the new method of adding deviations, you no longer have to add these flag values here. So these lines can be removed.